src/modules/RemediationsModal/RemediationsWizard.js

```diff
@@ -132,7 +132,8 @@ export const RemediationWizard = ({ setOpen, data, basePath, registry }) => {
   const validatorMapper = {
     'validate-systems': () => (value) =>
       value &&
-      Object.values(value).filter((value) => typeof value !== undefined).length
+      Object.values(value).filter((value) => typeof value !== 'undefined')
```

Comment:
new eslint rule does not allow `typeof value !== undefined`. The valid check now is: https://eslint.org/docs/latest/rules/valid-typeof#options

Reply:
Can you not do `.filter((value) => !!value)` here?